Assignments/SCF-Session-3/string/MyString.java

```diff
@@ -0,0 +1,209 @@
+package string;
+
+import java.util.Scanner;
+
+/**
+ * This class performs operations on strings without using Java APIs.
+ * @author Ishita Mathur
+ *
+ */
+public class MyString 
+{
+	
+	/**
+	 * Calculates length of string
+	 * @param string, string != null
+	 * @return length of string
+	 */
+	public int stringLength(String string)
+	{
+		string = string + '\0'; //add null to the given string
+		int count = 0;
+
+		for (int i = 0; string.charAt(i) != '\0'; i++) 
+		{
+			count++;
+		}
+
+		return count;
+	}
+	
+	/**
+	 * Compares two string for equality
+	 * @param string1 != null
+	 * @param string2 != null
+	 * @return returns 1 if strings are equal and 0 if unequal
+	 */
+	public int compareStrings(String string1, String string2) throws ArithmeticException
```

Comment:
`throws ArithmeticException` try and catch should be used to throw an exception.